ts/@live-compositor/node/src/fetch.ts

```diff
@@ -33,6 +34,28 @@ export async function sendRequest(baseUrl: string, request: ApiRequest): Promise
   return (await response.json()) as object;
 }
 
+export async function sendMultipartRequest(
+  baseUrl: string,
+  request: MultipartRequest
+): Promise<object> {
+  const response = await fetch(new URL(request.route, baseUrl), {
+    method: request.method,
+    body: request.body as FormData,
+    agent: url => (url.protocol === 'http:' ? httpAgent : httpsAgent),
+  });
+  if (response.status >= 400) {
+    const err: any = new Error(`Request to compositor failed.`);
+    err.response = response;
+    try {
```

Comment:
I think we talked about it already, if json() failes then `text()` will also fail because body was already read. This implementation and `sendRequest` is incorrect